activerecord/lib/active_record/relation/query_methods.rb

```diff
@@ -722,12 +722,28 @@ def left_outer_joins!(*args) # :nodoc:
     # === no argument
     #
     # If no argument is passed, #where returns a new instance of WhereChain, that
-    # can be chained with #not to return a new relation that negates the where clause.
+    # can be chained with #not, #missing, or #associated.
```

Comment:
Since these methods are defined on `WhereChain`, they will actually be rendered as unformatted text, rather than links or inline code.  (The current documentation is similarly broken: https://api.rubyonrails.org/v7.0.3/classes/ActiveRecord/QueryMethods.html#method-i-where.)

We can prefix them with `WhereChain`:

```suggestion
    # can be chained with WhereChain#not, WhereChain#missing, or WhereChain#associated.
```

Or we can use `rdoc-ref` links:

```suggestion
    # can be chained with {not}[rdoc-ref:WhereChain#not],
    # {missing}[rdoc-ref:WhereChain#missing], or
    # {associated}[rdoc-ref:WhereChain#associated].
```

The former are more verbose when rendered, but the latter are not rendered as inline code (i.e. no highlight, no monospace).  I think I lean towards the former.

Reply:
Drive-by thought inspired by the above comment, without having actually reviewed the PR: `"chained with #not"` -> `"chained as {where.not}[rdoc-ref..]"`? (I don't recall, but assume from context, that we can't get both monospace and a custom link in rdoc?)

Reply:
> I don't recall, but assume from context, that we can't get both monospace and a custom link in rdoc?

At least, I have not figured out how to do so.  From memory, the following didn't work:

* `{<tt>method_name</tt>}[rdoc-ref:...]`
* `{+method_name+}[rdoc-ref:...]`
* `<tt>{method_name}[rdoc-ref:...]</tt>` (=> rendered literally as code)
* `+{method_name}[rdoc-ref:...]+` (=> exceeds the limited support for quoting with `+`)

Reply:
Should we open an issue on rdoc for this? It seems like this would be pretty useful